src/frontend/Sider/Behandling/RevurderFra/TilsynBarn/VedtakshistorikkTilsynBarnTabellVisning.tsx

```diff
@@ -12,7 +15,7 @@ type Props = {
 };
 
 const VedtakshistorikkTilsynBarnTabellVisning: React.FC<Props> = ({ vedtakBarnetilsyn }) => {
-    if (!erVedtakInnvilgelse(vedtakBarnetilsyn)) return 'Ingen vedtakshistorikk';
+    if (!erVedtakInnvilgelseEllerOpphør(vedtakBarnetilsyn)) return 'Ingen vedtakshistorikk';
```

Comment:
I stedet for å sjekke om vedtaket _ikke_ er innvilgelse eller opphør, kunne vi heller bare sjekket om det er avslag?

Reply:
Done: 8f975ab885177e306c062a31b70fba2c2c6fe53b